src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerPanel.tsx

```diff
@@ -103,8 +105,7 @@ export const TemplatesCatalogDrawerPanel: FC<TemplatesCatalogDrawerPanelProps> =
         .catch((err) => {
           setError(err);
         });
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [ns, template]);
+    }, [vmNamespace, template]);
```

Comment:
This is also not good, you still need `// eslint-disable-next-line react-hooks/exhaustive-deps` here. No, we don't want to add `updateVM` to dependency array. IMO it was better to merge https://github.com/kubevirt-ui/kubevirt-plugin/pull/1505 instead.